server/test/repository/ApplicationStatusesRepositoryTest.java

```diff
@@ -39,16 +37,11 @@ public void canQueryForActiveApplicationStatuses() {
     Long uniqueProgramId = new Random().nextLong();
     ProgramModel program =
         ProgramBuilder.newActiveProgram("Active status tests" + uniqueProgramId, "description")
+            .withStatusDefinitions(new StatusDefinitions(ImmutableList.of(APPROVED_STATUS)))
             .build();
-    String programName = program.getProgramDefinition().adminName();
-    StatusDefinitions statusDefinitions = new StatusDefinitions(ImmutableList.of(APPROVED_STATUS));
-    ApplicationStatusesModel applicationStatusesModel =
-        new ApplicationStatusesModel(
-            programName, statusDefinitions, StatusDefinitionsLifecycleStage.ACTIVE);
-    applicationStatusesModel.save();
-    applicationStatusesModel.setCreateTimeForTest("2041-01-01T00:00:00Z").save();
     // test
```

Comment:
**nitpick:** Not related to your PR but this test comment seems like it was accidentally put there. Would you mind removing it?

Reply:
removed :)